client/src/modules/stock/exit/exit.js

```diff
@@ -6,141 +6,124 @@ StockExitController.$inject = [
+        width : 75,
+        displayName : 'TABLE.COLUMNS.UNIT',
+        headerCellFilter : 'translate',
+        cellTemplate : 'modules/stock/exit/templates/unit.tmpl.html',
+      }, {
+        field : 'available_lot',
+        width : 150,
+        displayName : 'TABLE.COLUMNS.AVAILABLE',
+        headerCellFilter : 'translate',
+        cellTemplate : 'modules/stock/exit/templates/available.tmpl.html',
+      }, {
+        field : 'amount',
+        width : 150,
+        displayName : 'TABLE.COLUMNS.AMOUNT',
+        headerCellFilter : 'translate',
+        cellTemplate : 'modules/stock/exit/templates/amount.tmpl.html',
+      }, {
+        field : 'expiration_date',
+        width : 150,
+        displayName : 'TABLE.COLUMNS.EXPIRE_IN',
+        headerCellFilter : 'translate',
+        cellTemplate : 'modules/stock/exit/templates/expiration.tmpl.html',
       },
-
-      {
-        field: 'unit_price',
-        width: 150,
-        displayName: 'TABLE.COLUMNS.UNIT_PRICE',
-        headerCellFilter: 'translate',
-        cellTemplate: 'modules/stock/exit/templates/price.tmpl.html'
-      },
-
-      {
-        field: 'quantity',
-        width: 150,
-        displayName: 'TABLE.COLUMNS.QUANTITY',
-        headerCellFilter: 'translate',
-        cellTemplate: 'modules/stock/exit/templates/quantity.tmpl.html',
-        treeAggregationType: uiGridGroupingConstants.aggregation.SUM
-      },
-
-      {
-        field: 'unit_type',
-        width: 75,
-        displayName: 'TABLE.COLUMNS.UNIT',
-        headerCellFilter: 'translate',
-        cellTemplate: 'modules/stock/exit/templates/unit.tmpl.html'
-      },
-
-      {
-        field: 'available_lot',
-        width: 150,
-        displayName: 'TABLE.COLUMNS.AVAILABLE',
-        headerCellFilter: 'translate',
-        cellTemplate: 'modules/stock/exit/templates/available.tmpl.html'
-      },
-
-      {
-        field: 'amount',
-        width: 150,
-        displayName: 'TABLE.COLUMNS.AMOUNT',
-        headerCellFilter: 'translate',
-        cellTemplate: 'modules/stock/exit/templates/amount.tmpl.html'
-      },
-
-      {
-        field: 'expiration_date',
-        width: 150,
-        displayName: 'TABLE.COLUMNS.EXPIRE_IN',
-        headerCellFilter: 'translate',
-        cellTemplate: 'modules/stock/exit/templates/expiration.tmpl.html'
-      },
-
-      { field: 'actions', width: 25, cellTemplate: 'modules/stock/exit/templates/actions.tmpl.html' },
+      { field : 'actions', width : 25, cellTemplate : 'modules/stock/exit/templates/actions.tmpl.html' },
     ],
-    onRegisterApi: onRegisterApi,
-    data: vm.Stock.store.data,
-    fastWatch: true,
-    flatEntityAccess: true,
+    data : vm.Stock.store.data,
+    fastWatch : true,
+    flatEntityAccess : true,
   };
 
   vm.gridOptions = gridOptions;
   vm.checkValidity = checkValidity;
 
-  // expose the API so that scrolling methods can be used
-  function onRegisterApi(api) {
-    vm.gridApi = api;
-  }
-
   // exit type
```

Comment:
This comment tells almost nothing, I know it was there before this P.R

Reply:
Ha! :+1: